.github/workflows/default.yml

```diff
@@ -19,27 +18,31 @@ jobs:
       max-parallel: 4
       matrix:
         molecule_distro:
-          - centos7
```

Comment:
this should be enabled back later. I would lat it be disabled for this PR

Reply:
> this should be enabled back later. I would lat it be disabled for this PR

most probably centos7 can't be enabled back because of outdated base image which doesn't support cgroups v2. the only solution here is to downgrade to base image based on cgroups v1 and does't change existing approach (as this was done [here](https://github.com/mrlesmithjr/ansible-mariadb-galera-cluster/pull/139/commits/156d6aa10e571f2711bdbbc6d278f1e66f64a3db)).

Reply:
I would rather lean toward newer technologies. I thought that @eRadical would add his opinion on this topic. I propose to merge it now.